charts/shoot-addons/charts/heapster/templates/_deploymentspec.yaml

```diff
@@ -18,6 +18,9 @@ spec:
       - key: CriticalAddonsOnly
         operator: Exists
       serviceAccountName: {{ template "fullname" . }}
+      securityContext:
+        runAsUser: 65534
```

Comment:
When disabling privileged access in a cluster then heapster cannot be scheduled:
`Error: container has runAsNonRoot and image will run as root`.